### ISSUE TITLE
Unexpected error in latest version

### ISSUE THREAD

**Issue by SadSatanSama** [Anime4000/IFME#206]
![Screenshot (164)](https://user-images.githubusercontent.com/37984514/89789250-f948b080-db3d-11ea-884c-dc2f57385c74.png)
IDK if it's a bug or something but this error comes across me for some specific encoded videos(dunno any special common characteristics for them) and miraculously doesn't appear for older version for the same video(I used v.7.7)
please fix this ASAP

**Comment by Anime4000** [comment]
oh, mkvmux in ffmpeg

**Comment by VirusVenomForever** [comment]
this problem got solved or should wait for fix or new version? I understand now hard time, but i wanna know(if it's not too hard to explain) is it have some solution to solve this problem?

**Comment by SadSatanSama** [comment]
this bug still persists on the latest 21.05.25 debug version.. kindly look into this senpai

**Comment by Anime4000** [comment]
same error are you getting?

**Comment by SadSatanSama** [comment]
yes senpai.. tried that new AV1 encoder thing also.. same result

**Comment by Anime4000** [comment]
wait for another update

**Comment by Anime4000** [comment]
@SadSatanSama I have release 21.06.13, it should fix this issue
https://sourceforge.net/projects/ifme/files/encoder-gui/21.x/IFME-21.06.13_Debug.7z/download

give a try, see if works

**Comment by VirusVenomForever** [comment]
yeah, it fixed - thanks!

**Comment by Anime4000** [comment]
allright, I close this issue, reopen if got problem :)

**Comment by SadSatanSama** [comment]
hi, sorry for reporting late as I was out of station.. thing is I tested it with 2 different processors( ryzen 3 2200G and ryzen 5 5600x) works fine in the 2nd one.. so thanks for solving the issue.. but when tried in the first one the encoding process got stuck after like 80% completion(for 2.5 hrs) so this ongoing issue is solved. but I'll like to inform you that the program got much less optimized now even for ryzen.

**Comment by Anime4000** [comment]
> 
> 
> hi, sorry for reporting late as I was out of station.. thing is I tested it with 2 different processors( ryzen 3 2200G and ryzen 5 5600x) works fine in the 2nd one.. so thanks for solving the issue.. but when tried in the first one the encoding process got stuck after like 80% completion(for 2.5 hrs) so this ongoing issue is solved. but I'll like to inform you that the program got much less optimized now even for ryzen.

Sorry to hear that, I use Ryzen 3950X to compile, with strict `-march=znver2` optimization.
You are using `x265 MultiCoreWare (AVX2, GCC 10.2.0)` on both processor right?

**Comment by SadSatanSama** [comment]
> > hi, sorry for reporting late as I was out of station.. thing is I tested it with 2 different processors( ryzen 3 2200G and ryzen 5 5600x) works fine in the 2nd one.. so thanks for solving the issue.. but when tried in the first one the encoding process got stuck after like 80% completion(for 2.5 hrs) so this ongoing issue is solved. but I'll like to inform you that the program got much less optimized now even for ryzen.
> 
> Sorry to hear that, I use Ryzen 3950X to compile, with strict `-march=znver2` optimization.
> You are using `x265 MultiCoreWare (AVX2, GCC 10.2.0)` on both processor right?

yeah.. of course

**Comment by Anime4000** [comment]
I checked that Ryzen 3 2200G using first gen Zen Arch
https://en.wikichip.org/wiki/amd/ryzen_3/2200g

probably need generic arch for this

**Comment by SadSatanSama** [comment]
> I checked that Ryzen 3 2200G using first gen Zen Arch
> https://en.wikichip.org/wiki/amd/ryzen_3/2200g
> 
> probably need generic arch for this

so is it solvable with keeping other features intact? or do I need to use my laptop(ryzen 5) from now on?? because I really don't wanna use a laptop for encoding ;___;

**Comment by Anime4000** [comment]
> 
> 
> > I checked that Ryzen 3 2200G using first gen Zen Arch
> > https://en.wikichip.org/wiki/amd/ryzen_3/2200g
> > probably need generic arch for this
> 
> so is it solvable with keeping other features intact? or do I need to use my laptop(ryzen 5) from now on?? because I really don't wanna use a laptop for encoding ;___;

Try disable encoders, navigate to `Plugins\x265` and remove underscore of `_plugin.video1-amd64_avx2.json`
This will remove AVX2 encoder support
